packages/utils/index.js

```diff
@@ -1,26 +1,22 @@
 // @flow
 
 // Helpers
-export { capitalize } from './helpers/capitalize';
+export { capitalize } from './formatters/Capitalize';
 
 // Localization
 export { formatPrice } from './localization/CurrencyFormatter';
 export type { Price } from './localization/CurrencyFormatter';
-export { default as DateFormatter } from './localization/DateFormatter';
-export { default as DateUtils } from './localization/DateUtils';
 export { default as Device } from './device/Device';
 
 // LayoutContext
-export { LAYOUT } from './layoutContext/LayoutConstants';
-export type { LayoutContextState } from './layoutContext/LayoutContext';
-export { withLayoutContext } from './layoutContext/LayoutContext';
-export {
-  default as LayoutContextProvider,
-} from './layoutContext/LayoutContext';
+export { LAYOUT } from './context/LayoutConstants';
+export type { LayoutContextState } from './context/LayoutContext';
+export { withLayoutContext } from './context/LayoutContext';
+export { default as LayoutContextProvider } from './context/LayoutContext';
 
 // src
-export { default as withContext } from './src/withContext/withContext';
-export { default as formatDate } from './src/formatDate';
+export { default as withContext } from './context/withContext';
```

Comment:
We should keep `LayoutContext` and `withContext` in separate subfolders.

Also, I know that I've suggested it as possible structure in description. But now when I see it I'll be more for something slightly more universal. I think it can be moved directly into `helpers`.

Or maybe even better we can consider moving everything into `src` subfolder (into separate subfolders). Because we don't have that many utils (looks like 7 for now) and border between packages is sometimes quite thin (for example `FormatDate` in `formatters`, but `CurrencyFormatter` in `localization` can be confusing) and it can be often hard to choose where new util should be put. I'll let it up to you.

Reply:
Moved everything to `src` @JoseDuSV